components/developer/distcc/Makefile

```diff
@@ -0,0 +1,122 @@
+
+COMPONENT_SUMMARY=	distcc - distributor of preprocessed C/C++/ObjC files across nodes to crowd-build code (bleeding edge, Git $(COMPONENT_GIT_REV) at $(COMPONENT_GIT_DATE))
+COMPONENT_FMRI=		developer/distcc
+COMPONENT_CLASSIFICATION=	Development/C
+COMPONENT_LICENSE=	"GPLv2"
+COMPONENT_LICENSE_FILE=	COPYING
+
+include $(WS_MAKE_RULES)/prep.mk
+include $(WS_MAKE_RULES)/configure.mk
+include $(WS_MAKE_RULES)/ips.mk
+
+# distcc (pump) requires python3...
+PYTHON_VERSION=3.4
+
+COMPONENT_PREP_ACTION = \
+    ( cd $(@D) && ./autogen.sh )
+
+COMPONENT_PRE_CONFIGURE_ACTION =	($(CLONEY) $(SOURCE_DIR) $(@D))
+
+COMPONENT_POST_INSTALL_ACTION =	( cd $(PROTOUSRSHAREMAN1DIR) && for F in *.gz ; do [ -s "$$F" ] && gzip -cd < "$$F" > `basename "$$F" .gz` ; done )
+
+# Find the <config.h> as included by some sources - must be before GCC plugin dir
+CFLAGS += -I$(BUILD_DIR_$(BITS))/src
+CXXFLAGS += -I$(BUILD_DIR_$(BITS))/src
+CPPFLAGS += -I$(BUILD_DIR_$(BITS))/src
+
+# Where to look for libiberty header and .a file: the GCC p5m manifest
+# is explicit about the paths - down to i386'iness of them. Cool :\
+# NOTE: Match this with GCC_ROOT, currently referring to "4.9"
+CFLAGS += -I$(GCC_ROOT)/lib/gcc/i386-pc-solaris2.11/4.9.3/plugin/include -L$(GCC_ROOT)/lib
+
+CXXFLAGS+=	$(CPP_LARGEFILE)
+CFLAGS +=	$(CPP_LARGEFILE)
+
+CFLAGS +=	-D_GNU_SOURCE -D__EXTENSIONS__
+CXXFLAGS +=	-D_GNU_SOURCE -D__EXTENSIONS__
+
+CXXFLAGS +=	-D_POSIX_PTHREAD_SEMANTICS
+CFLAGS +=	-D_POSIX_PTHREAD_SEMANTICS
+
+COMPONENT_INSTALL_ARGS +=	DESTDIR="$(PROTO_DIR)"
+COMPONENT_INSTALL_ARGS +=	PYTHON="$(PYTHON)"
+
+CONFIGURE_ENV +=	PYTHON="$(PYTHON)"
+INSTALL_ENV   +=	PYTHON="$(PYTHON)"
+
+CONFIGURE_OPTIONS +=	--sysconfdir=$(ETCDIR)
+
+# use getaddrinfo, getnameinfo, etc :
+CONFIGURE_OPTIONS +=	--enable-rfc2553
+
+# GSS-API mutual auth
+# NOTE: May require some source chizeling, skip in first recipe iterations
+REQUIRED_PACKAGES +=	system/kernel/security/gss system/library/security/gss service/security/gss system/header
+CFLAGS += -Wno-error=unused-but-set-variable
+CONFIGURE_OPTIONS +=	--with-auth
+#CONFIGURE_OPTIONS +=	--without-auth
+
+# Support mDNS "zeroconf" peer discovery/announcement
+# NOTE: May require some source chizeling, skip in first recipe iterations
+#CONFIGURE_OPTIONS +=	--with-avahi
+CONFIGURE_OPTIONS +=	--without-avahi
+
+# GUIs for monitoring
+CONFIGURE_OPTIONS +=	--with-gnome
+CONFIGURE_OPTIONS +=	--with-gtk
+
+# common targets
+# NOTE: Currently GCC libiberty is only published as a 32-bit .a file
+# so we can only build a 32-bit distcc
+build:          $(BUILD_32)
+
+install:        $(INSTALL_32)
+
+test:           $(NO_TESTS)
+
+# Auto-generated contents below.  Please manually verify and remove this comment
+REQUIRED_PACKAGES += library/desktop/gtk2
+REQUIRED_PACKAGES += library/desktop/libgnome
+REQUIRED_PACKAGES += library/desktop/libgnomeui
+REQUIRED_PACKAGES += library/glib2
+REQUIRED_PACKAGES += library/popt
+REQUIRED_PACKAGES += runtime/python-26
```

Comment:
BTW, if there's choice between Python 2.6 and 2.7, use 2.7 (we are slowly going to make it the default one). If we already require python 3.4 and everything works with python 3.4, it would be good to use just one python version